app/views/pages/home.html.erb

```diff
@@ -86,7 +86,7 @@
 				  <div class="card-body">
 				    <span class="card-title"><%= highlight.title %></span>
 				    <p class="card-text"><%= highlight.blurb %></p>
-				    <%= link_to "See more >", highlight.link, class: 'see-more' %>
+				    <%= link_to "See more >", highlight_path(highlight), class: 'see-more' %>
```

Comment:
Just making sure this requirement changed. We now want to go to the show page for each highlight, and not to the link that was included when the highlight was created?

Reply:
Cynthia was insistent about this.